.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,23 @@
+name: Node CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ubuntu-latest, windows-latest, macOS-latest]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - name: build it
+        run: |
+          npx envinfo
+          cmake .
```

Comment:
@danbev Should I change this to:
```bash
mkdir build
cd build
cmake ..
cmake --build .
```

Reply:
Also do I need to delete `Makefile`

Reply:
I don't think it matters on the CI system (having the generated files mixed with the sources is more a preference when running locally I think.